### PR TITLE
Add phpcs checks

### PR DESCRIPTION
**Pull request by samwilson**
This adds codesniffer checks to Travis. The code does not yet pass; I just wanted to send this PR to see what you think? I was confused about what you were saying about using PSR0 — does that mean you're also keen to move to the other PSR coding standards? Alternatively, you could update your in-house coding styles to inherit from PSR2.

### CODE REVIEW

**Comment by TheFox** [comment]
My fault. You are right. PSR-0 is autoloading. I meant PSR-1. Or maybe PSR-2. I'm always using the Auto Format option in PHPStorm. This is just a keyboard shortcut.

`thefox/phpcsrs` is not PSR compatible. Maybe using PHPCS directly without `thefox/phpcsrs` is better. But I don't know if this is working yet.

**Comment by samwilson** [comment]
Ah, cool, okay: updated with PSR2. I'll fix the outstanding exceptions.

**Comment by samwilson** [comment]
Okay, all passing now. Have a look at the 2nd commit for the non-automatic changes. There aren't any in either of these commits that make any changes to functionality or logic.

File: src/TheFox/FlickrCli/Command/UploadCommand.php [hunk 1]
```diff
@@ -76,11 +76,12 @@ protected function configure()
         $this->setName('upload');
         $this->setDescription('Upload files to Flickr.');
 
+        $csvDesc = 'Comma separated names. For example: --sets=set1,set2';
         $this->addOption('config', 'c', InputOption::VALUE_OPTIONAL, 'Path to config file. Default: config.yml');
         $this->addOption('log', 'l', InputOption::VALUE_OPTIONAL, 'Path to log directory. Default: log');
         $this->addOption('description', 'd', InputOption::VALUE_OPTIONAL, 'Description for all uploaded files.');
-        $this->addOption('tags', 't', InputOption::VALUE_OPTIONAL, 'Comma separated names. For example: --tags=tag1,tag2');
-        $this->addOption('sets', 's', InputOption::VALUE_OPTIONAL, 'Comma separated names. For example: --sets=set1,set2');
+        $this->addOption('tags', 't', InputOption::VALUE_OPTIONAL, $csvDesc);
```
Comment (TheFox) [inline src/TheFox/FlickrCli/Command/UploadCommand.php]: The string in `$csvDesc` is not valid for option `tags`. There is a separate example for `tags` and `sets`.

Comment (samwilson) [reply src/TheFox/FlickrCli/Command/UploadCommand.php]: agh, yes! how foolish of me. sorry; fixed now.

**Pull request merged**